### PR TITLE
CS fixes for plugin folder

### PR DESCRIPTION
**Pull request by simon-letch**

### CODE REVIEW

File: Plugin/RuleCollectionPlugin.php [hunk 1]
```diff
@@ -21,17 +21,16 @@ class RuleCollectionPlugin
...
     }
 
     /**
      * @param \Magento\SalesRule\Model\ResourceModel\Rule\Collection $subject
-     * @param $result
-     * @param $websiteId
-     * @param $customerGroupId
-     * @param $couponCode
+     * @param mixed $result
+     * @param mixed $websiteId
```
Comment (adeelq) [inline Plugin/RuleCollectionPlugin.php]: should be int

File: Plugin/RuleCollectionPlugin.php [hunk 2]
```diff
@@ -21,17 +21,16 @@ class RuleCollectionPlugin
...
+     * @param mixed $customerGroupId
```
Comment (adeelq) [inline Plugin/RuleCollectionPlugin.php]: should be int

File: Plugin/RuleCollectionPlugin.php [hunk 3]
```diff
@@ -21,17 +21,16 @@ class RuleCollectionPlugin
...
+     * @param mixed $couponCode
```
Comment (adeelq) [inline Plugin/RuleCollectionPlugin.php]: should be string

File: Plugin/RuleCollectionPlugin.php [hunk 4]
```diff
@@ -45,11 +44,14 @@ public function afterSetValidationFilter(
...
         if (! empty($couponCode)) {
-            $select->where('(rule_coupons.expiration_date IS NULL) AND
-                     (to_date is null or to_date >= ?)
-                    OR
-                     (rule_coupons.expiration_date IS NOT NULL) AND
-                     (rule_coupons.expiration_date >= ?) ', $now);
+            $select->where(
+                '(rule_coupons.expiration_date IS NULL) AND
+                (to_date is null or to_date >= ?)
+                OR
```
Comment (adeelq) [inline Plugin/RuleCollectionPlugin.php]: Please move OR in to line 49. Just like AND in line 48. Much better to read then

File: Plugin/TransportPlugin.php [hunk 5]
```diff
@@ -41,7 +41,8 @@ public function __construct(
         }
         $this->message  = $message;
         $this->helper   = $helper;
-        $this->smtp = $smtpFactory->create([
+        $this->smtp = $smtpFactory->create(
+            [
             'host' => $this->helper->getSmtpHost(),
```
Comment (adeelq) [inline Plugin/TransportPlugin.php]: indent please

File: Plugin/TransportPlugin.php [hunk 6]
```diff
@@ -41,7 +41,8 @@ public function __construct(
...
             'config' => $this->helper->getTransportConfig()]
```
Comment (adeelq) [inline Plugin/TransportPlugin.php]: indent please and close array ] on new line

Comment (simon-letch) [inline Plugin/TransportPlugin.php]: Done

File: Plugin/TransportPlugin.php (hunk 5 above)
Comment (simon-letch) [inline Plugin/TransportPlugin.php]: Don't think we can indent this as phpcs will reject indentation? Can you check in PHPStorm?

Comment (adeelq) [inline Plugin/TransportPlugin.php]: leave it then.

**Pull request merged**